### ISSUE TITLE
Change top level element from graphs to unique id of model and add version info

### ISSUE THREAD

**Issue by pgleeson** [ModECI/MDF#19]
In line with #18, I propose to make the top level entry of the specification the id of the model, referencing 1) the graph(s) it contains and 2) the version of the language used:
```
{
    "Simple": {
        "format": "ModECI MDF v0.1",
        "graphs": {
            "simple_example": {
                 ...
           }
     }
}
```

This is in keeping with the proposal in #18 that everything with an id is referenced in the same way, and the fact that models can in theory contain multiple graphs, and so one overall id should be used for the "model" as opposed to its constituent graph(s).

**Comment by pgleeson** [comment]
Will be closed by #59

**Comment by pgleeson** [comment]
Fixed in code and main readme
